src/MicrosoftTeams.schema.json

```diff
@@ -92,17 +98,20 @@
         },
         "accentColor": {
             "type": "string",
-            "description": "A color to use in conjunction with the icon. The value must be a valid HTML color code starting with '#', for example `#4464ee`."
+            "description": "A color to use in conjunction with the icon. The value must be a valid HTML color code starting with '#', for example `#4464ee`.",
+            "pattern": "^#[0-9a-fA-F]{6}$"
         },
         "galleryTabs": {
             "type": "array",
             "description": "The set of tabs that users can optionally add to Teams",
+            "maxItems": 1,
             "items": {
                 "type": "object",
                 "properties": {
                     "configurationUrl": {
                         "type": "string",
-                        "description": "The url to use when configuring the tab."
+                        "description": "The url to use when configuring the tab.",
+                        "maxLength": 2048
```

Comment:
I wonder if we can do some basic URL validation using regex here. Nothing as complex at what we use in the FE client but at least a check that the URL starts with https: would be great.